src/coreclr/nativeaot/Runtime.Base/src/System/AttributeUsageAttribute.cs

```diff
@@ -29,7 +29,7 @@ public AttributeUsageAttribute(AttributeTargets validOn, bool allowMultiple, boo
         // Allowing the set properties as it allows a more readable syntax in the specifiers (and are commonly used)
         // The get properties will be needed only if these attributes are used at Runtime, however, the compiler
         // is getting an internal error if the gets are not defined.
-
+#pragma warning disable CA1822
```

Comment:
Runtime.Base is not System.Private.CoreLib. it's the runtime library where lowest levels of the runtime are implemented. It has its own System.Object. We might be able to get rid of some of this, but maybe not in this pull request. AttributeUsage doesn't sound like something necessary unless it's forced by Roslyn. Some duplication is necessary because we don't want Runtime.Base to become CoreLib. Some of the duplication is just historic artifacts.